demos/playingfield/ceo.py

```diff
@@ -0,0 +1,99 @@
+import rospy
+
+from pycram.worlds.bullet_world import BulletWorld
+from pycram.datastructures.enums import ObjectType, WorldMode
+from pycram.world_concepts.world_object import Object
+from pycram.datastructures.pose import Pose
+
+from pycram.datastructures.dataclasses import Color
+from demos.playingfield.sandcastle import generic_plan
+from pycram import World
+from pycram.failures import PlanFailure
+from pycram.tasktree import task_tree
+
+import traceback
+import sqlalchemy
+import pycram.orm.base
+from pycram.external_interfaces.procthor import ProcThorInterface
+
+# Many Plans in many environments with many robots
+# Don't know where to store NEEM so I start with the memory db
```

Comment:
If it is a demo remove shitty Statements Like this

Reply:
Done